backend/backend/common/message.py

```diff
@@ -0,0 +1,34 @@
+from pydantic import BaseModel, validator
+
+# This module describes how we want a message will appear on our websocket.
+
+VERSION = '1'
+
+# Once a more standard content will be made,
+# every content-type will inherit from abstract class.
```

Comment:
Is that a todo?

Reply:
it more of an aknolwlagment that this is a work-in-progress that depends on frontend implmentation